docs/src/operate/troubleshooting/monitoring-service-health.md

```diff
@@ -37,7 +37,7 @@ For example, `tedge-mapper-c8y` publishes a message on topic `te/device/main/ser
 |----------|----------------------------------------------------|
 | `pid`    | Process ID of the service                          |
 | `status` | Service status. Possible values are `up` or `down` |
-| `time`   | Timestamp in RFC3339 format                        |
+| `time`   | Timestamp in Unix time format                      |
```

Comment:
Timestamp in the  time format specified by `tedge config` for `service.timestamp_format`: either  `unix` or `rfc-3339`.